cfme/common/vm.py

```diff
@@ -568,8 +568,10 @@ class VM(BaseVM):
 
     retire_form = Form(fields=[
         ('date_retire',
-            AngularCalendarInput("retirement_date",
-                                 "//label[contains(normalize-space(.), 'Retirement Date')]")),
+            AngularCalendarInput(version.pick(
+                {"5.9": "retirement_date_datepicker",
+                version.LOWEST: "retirement_date"}),
+                "//label[contains(normalize-space(.), 'Retirement Date')]")),
```

Comment:
**REQUIRED**: This introduces an import-time appliance access. remove the version.pick and instead of that, set the whole value as the version pick dict with the two AngularCalendarInput definitions. Old webui system can detect and process dictionaries as version picks in known places like this.